kayak_core/src/styles/mod.rs

```diff
@@ -300,6 +300,8 @@ define_styles! {
         pub top: StyleProp<Units>,
         /// The width of this widget
         pub width: StyleProp<Units>,
+        /// The font style of the widget
```

Comment:
Might want to clarify that this expects the font name.
```suggestion
        /// The font name of the widget
        ///
        /// Only applies to [`RenderCommand::Text`]
```